### ISSUE TITLE
Fullscreen transparent window is not transparent

### ISSUE THREAD

**Issue by gnomesley** [electron/electron#8439]
* Electron version: 1.4.14
* Operating system: Windows 8.1

### Expected behavior

A fullscreen transparent window should be transparent.

### Actual behavior

The window is not transparent (has a white background).

### How to reproduce

```
electron.app.on("ready", function () {
    var win = new BrowserWindow({
        show: true,
        frame: false,
        fullscreen: true,
        transparent: true
    });
});
```

Refs https://github.com/electron/electron/issues/8321#issuecomment-273615701

**Comment by kevinsawicki** [comment]
Thanks, got #8438 going to address this.

**Comment by kevinsawicki** [comment]
Fixed by #8438
